tests/runtime/test_http_server.py

```diff
@@ -215,11 +215,11 @@ def test_docs():
         assert response.status_code == 200
 
 
-def test_inference(sample_task_model_id):
+def test_inference_sample_task(sample_task_model_id):
     """Simple check that we can ping a model"""
     server = http_server.RuntimeHTTPServer()
     with TestClient(server.app) as client:
-        json_input = {"inputs": {"sample_input": {"name": "world"}}}
+        json_input = {"inputs": {"name": "world"}}
```

Comment:
Do we have a test that checks that `{"inputs": {"sample_input": {"name": "world"}}}` gives a 400?

Reply:
I thought about that - the issue is that the above doesn't give a 400 since it's technically not an incorrect request right? The only thing that happens is that the arg `sample_input` in the `run` function gets a `None` value passed to it

Reply:
Something like `json_input = {"blah": {"sample_input": {"name": "world"}}}` would give a 400, I can add a test for that

Reply:
> the issue is that the above doesn't give a 400 since it's technically not an incorrect request right? The only thing that happens is that the arg sample_input in the run function gets a None value passed to it

hmm....  I'd say it is an invalid request since every request should have all of the required parameters set, and those should be correctly passed along to the model. If we ever try to run a model with a required parameter set to `None`, then we've messed up somewhere